app/Http/Controllers/NoteController.php

```diff
@@ -29,6 +30,40 @@ public function index(Request $request)
         ]);
     }
 
+    public function getNotesByUsername()
+    {
+        $user = Auth::user();
+
+        if (!$user) {
+            return response()->json(['message' => 'User not logged in.'], 401);
+        }
+
+        return response()->json([
+            'notes' => User::with('notes')->find($user->id)['notes'],
```

Comment:
When user is already loaded you are loading him again? Use `$user->notes`. If it doesn't work, your relationship on `User` or `Note` model is wrong